src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -238,6 +216,95 @@ private void initializeAppsLocation(CommonLoggerI log, File serverXML, File conf
         }
     }
 
+
+    /**
+     * jvm.options file read order
+     *   1. {wlp.user.dir}/
+     *   2. {server.config.dir}/configDropins/defaults/
+     *   3. {server.config.dir}/
+     *   4. {server.config.dir}/configDropins/overrides/
+     * @throws FileNotFoundException
+     * @throws Exception
+     */
+    public void processJvmOptions() throws FileNotFoundException, Exception {
+        final String jvmOptionsString = "jvm.options";
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.WLP_USER_DIR), jvmOptionsString));
+        parsePropertiesFromFile(getFileFromConfigDirectory(CONFIGDROPINS_DEFAULT + File.separator + jvmOptionsString));
```

Comment:
So I started wondering if configDirectory is the one specified in the pom.xml (or the default src/main/liberty/config), or if it should just be `server.config.dir` at this point? I looked at what LMP/LGP are passing in to this class, and I think it is what was configured in the build file. But it seems like we should just be concerned with the config files that got copied into the server itself at this point, in which case we could just use `libertyDirectoryPropertyToFile.get(ServerFeatureUtil.SERVER_CONFIG_DIR)` as the basis for the configDropins. Let me know if you didn't follow all of that.

Reply:
I think this could also solve a problem we have right now that we are not necessarily passing in all the right info for bootstrap props and server env (because of overriding Maven props for example), but if we used what got copied/generated in the server it should all be correct at that point.